wcvendors/wcvendors
# Fixed: Order status not getting updated while initiating a part refun…

…d. #827

### All Submissions:






### Changes proposed in this Pull Request:



Closes #827 .

### How to test the changes in this Pull Request:

1. Run an order
2. Run manually refund for that order item
3. Check commissions table and see both admin and vendor commission status was reversed

### Other information:
